### ISSUE TITLE
Translator login and registration

### ISSUE THREAD

**Issue by kmay90** [magnolo/newhere#155]
It would be great if a dedicated registration option for translators could be implemented where they only have to fill in relevant fields e.g. name and contact email, rather than address, as is currently the case

**Comment by ScrambledRK** [comment]
related to: Info need from users/NGOs #179?

**Comment by kmay90** [comment]
Yes, indeed related. One part of issues is not collecting unnecessary info from users, one way to deal with this would be to have a separate form for translators where less info is requested
